# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../popt/glibc2.2.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./popt.h.orig	Sat Oct  6 14:12:32 2001
+++ ./popt.h	Sat Oct  6 14:13:16 2001
@@ -10,6 +10,7 @@
 #endif
 
 #include <stdio.h>			/* for FILE * */
+#include <values.h>			/* for FLT_MIN/MAX */
 
 #define POPT_OPTION_DEPTH	10
 
